hrui/tests/phpunit/HRUITrait.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * A trait with helper methods to be reused among this extension's tests
+ */
+trait HrUITrait {
+
+  /**
+   * Creates single (Individuals) contact from the provided data.
+   *
+   * @param array $params should contain first_name and last_name
+   * @return int return the contact ID
+   * @throws \CiviCRM_API3_Exception
+   */
+  protected function createContact($params) {
```

Comment:
Now we have this exactly same method in three different traits. I wish there was a better way to share code among the extensions 😕